include/deal.II/grid/grid_generator.h

```diff
@@ -77,6 +77,26 @@ namespace GridGenerator
                    const bool                    colorize= false);
 
   /**
+   * \brief Mesh of a d-simplex with (d+1) vertices and mesh cells, resp.
+   *
+   * The <code>vertices<code> argument contains a vector with all d+1
+   * vertices of the simplex. They must be given in an order such that
+   * the first forms the origen of a set of vectors with positive
```

Comment:
I also don't understand that sentence. How does the the first point "form an origin"?